pkgpanda/__init__.py

```diff
@@ -261,6 +270,19 @@ def validate_compatible(packages, roles):
                         k, v, package))
             environment[k] = package
 
+        # No conflicting sysctl values.
+        for service_name, sysctl_settings in package.sysctl.items():
+            for sysctl_var, sysctl_value in sysctl_settings.items():
+                if sysctl_map.get(sysctl_var, None) == sysctl_value:
```

Comment:
Why not do:
```
if sysctl_var in sysctl_map:
  throw ValueError(...)
```

Practically we should be able to keep the setting in a single package (keeping multiple packages in work also sounds like more work than just having people update the one package it lives in).